targeted/search cvedetails using target server header.js

```diff
@@ -0,0 +1,13 @@
+// Captures Server header from the application response and searches cvedetails.com for known target server vulnerabilities.
+
+
+function invokeWith(msg) {
+	host = msg.getRequestHeader().getURI().getHost(); 
+if (msg) 
+	{
+		headers = msg.getResponseHeader().getHeaders("Server")
```

Comment:
The method ```getHeaders(String)``` returns a list of headers (if you just want one you should use ```getHeader(String)```, which returns a ```String```). If you really want to check for several ```Server``` headers it's probably better to create the query parameter's value manually (i.e. concatenate the values of the list) otherwise the value of ```q``` ends up with chars ```[``` and ```]``` (added because of conversion from list to string).
In both cases the return value is not defined if no ```Server``` header is present.